### PR TITLE
Add event organizer

### LINKED ISSUE #252: Add organizer in Version 2.*
The addOrganizer method has been depreciated in the latest version. Will it be added again?

### PR DESCRIPTION
**Pull request by markuspoerschke**
Fix #252

### CODE REVIEW

File: website/docs/component-event.md [hunk 1]
```diff
@@ -209,6 +210,30 @@ $event = new Event();
 $event->setLocation($location);
 ```
 
+### Organizer
+
+The Organizer defines the person who organises the event.
+The property consists of at least an email address.
```
[the remark this answers is not in the document]
Comment (markuspoerschke) [reply website/docs/component-event.md:216]: False finding...

**Comment by aarongerig** [comment]
@markuspoerschke Would it be possible to get a new version for that? :)

**Comment by markuspoerschke** [comment]
@aarongerig Thanks for the reminder.

My bot just created a new release: https://github.com/markuspoerschke/iCal/releases/tag/2.2.0

**Comment by aarongerig** [comment]
@markuspoerschke Awesome, thank you!

File: src/Domain/ValueObject/EmailAddress.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    public function getEmailAddress(): string
+    {
+        return $this->emailAddress;
+    }
+
+    public function toUri(): Uri
+    {
+        return new Uri('mailto:' . urlencode($this->emailAddress));
```
Comment (marcusmarch) [inline src/Domain/ValueObject/EmailAddress.php:36]: Outlook 2016 is having trouble with the encoded "@" character in the email address.

According to RFC standard, the `mailto:` doesn't require the url-encoding of that character. From the [RFC 5545 3.8.4.3 (Organizer)](https://www.rfc-editor.org/rfc/rfc5545#section-3.8.4.3) section, see its examples, but it says "Value Type: CAL-ADDRESS" -> look for that in the RFC, find [definition here](https://www.rfc-editor.org/rfc/rfc5545#section-3.3.3) -> says "the value must be a mailto URI, as defined by [[RFC2368](https://www.rfc-editor.org/rfc/rfc2368)].", see its examples.

By RFC 2368's examples, it seems that it implies the "@" should remain unencoded. The solution then would mean parsing the email address, such as is seen in this related StackOverflow discussion on the topic: https://stackoverflow.com/questions/8940445/what-is-the-correct-way-to-escape-a-string-for-a-mailto-link

In our case, we're going to just look for `%40` in the `ORGANIZER` property and convert it back to `@`, which will suffice.

**Pull request merged**